apertium-spa.spa.dix

```diff
@@ -19589,6 +19590,7 @@
 <e lm="usurpador">       <i>usurpador</i><par n="abrumador__adj"/></e>
 <e lm="usurpatorio">     <i>usurpatori</i><par n="absolut/o__adj"/></e>
 <e lm="uterino" a=" alseoane@gmail.com"><i>uterin</i><par n="absolut/o__adj"/></e>
+<e lm="utilísimo">		 <i>utilísim</i><par n="absolut/o__adj"/e></e>
```

Comment:
Superlatives go usually with the same root. I have added `utilísimo` here: https://github.com/apertium/apertium-spa/commit/012f456c09d16ecf5433826b12c941e03ae1abb2#diff-774c9ef6889e34c002c334a6cdd312b0R19813